1-select-tasks/15.txt

```diff
@@ -1,3 +1,21 @@
 Create a SQL query to return the total number of orders for every month in 1997 year:
 
-| January | February | March | April | May | June | July | August | September | November | December |
\ No newline at end of file
+| January | February | March | April | May | June | July | August | September | November | December |
+
+USE [Northwind]
+;
+
+SELECT
+(SELECT COUNT(OrderID) FROM Orders WHERE OrderDate BETWEEN ('19970101') AND ('19970131')) AS January
```

Comment:
Use Case https://docs.microsoft.com/en-us/sql/t-sql/language-elements/case-transact-sql. You can nest it in agregate funcitons.

Reply:
Also add solution with https://technet.microsoft.com/en-us/library/ms177410(v=sql.105).aspx